apps/tests_files/Matrix_Mul/Matrix_Mul.c

```diff
@@ -0,0 +1,50 @@
+void multiply(int mat1[][3], int mat2[][3], int res[][3])
```

Comment:
Add header to the file.
- short description.
- Who is the owner of this test.
- when was this test created
- comment and Explain the memory regions in this test